src/containers/Preview/PlaylistPreview/components/NextLink.js

```diff
@@ -77,7 +77,11 @@ function NextLink(props) {
                   })
                     .then((result) => {
                       if (result.value) {
-                        history.push(`/project/${projectId}/${shared ? 'preview' : 'preview'}`);
+                        if (showLti) {
+                          history.push(`/project/${projectId}/shared`);
+                        } else {
+                          history.push(`/project/${projectId}/${shared ? 'preview' : 'preview'}`);
```

Comment:
Ternary operator looks weird here
Is it same 'preview' whether shared is true or false?

Reply:
updated